app/controllers/posts_controller.rb

```diff
@@ -9,4 +11,8 @@ def show
     PostViewsService.new(@post, current_user).perform
     authorize @post
   end
+
+  def user_form_params
+    params.require(:user_form).permit(:first_name, :last_name, :role)
```

Comment:
Possible security vulnerability: Potentially dangerous key allowed for mass assignment